data/sea/54-zebra.h

```diff
@@ -154,68 +189,164 @@ static ierror_msg_t zebra_translate
     return 0;
 }
 
-static int64_t zebra_translate_table
+/* Translate Zebra enum tags to Icicle bool flags, e.g. is_some, is_right, etc. */
+static uint64_t zebra_translate_column_tags
     ( anemone_mempool_t *mempool
-    , iint_t elem_start
-    , iint_t count
+    , int64_t elem_start
+    , int64_t elem_count
+    , int64_t tag_column_count
     , void **dst
-    , const zebra_table_t *table )
+    , const int64_t *src )
 {
-    int64_t         offset = 0;
-    int64_t         cols_count  = table->column_count;
-    zebra_column_t *cols = table->columns;
-
-    //printf ("zebra_translate_table: elem_start = %d, count = %d\n", elem_start, count);
-    for (int64_t i = 0; i != cols_count; ++i) {
-        //printf ("zebra_translate_table: offset = %d\n", offset);
-        offset += zebra_translate_column (mempool, elem_start, count, dst + offset, cols + i);
+    /* enum B = F | T
+       ==>
+       bool is_true
+       ==>
+       F == tag 0 == 0
+       T == tag 1 == 1
+
+       enum E = A | B | C | D | E
```

Comment:
I guess I was wondering if we should reject zebra schemas which have sum types with more than two variants for the time being? If we ever added full blown algebraic data types to icicle, I think we'd have to change the runtime representation to tags at that time.